pkg/validation/options.go

```diff
@@ -28,11 +30,22 @@ import (
 // are of the correct format
 func Validate(o *options.Options) error {
 	if o.SSLInsecureSkipVerify {
-		// TODO: Accept a certificate bundle.
 		insecureTransport := &http.Transport{
 			TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
 		}
 		http.DefaultClient = &http.Client{Transport: insecureTransport}
+	} else if o.ProviderCAFiles != nil && len(o.ProviderCAFiles) > 0 {
+		pool, err := util.GetCertPool(o.ProviderCAFiles)
+		if err != nil {
+			return err
```

Comment:
Maybe we should wrap this to add more context
```suggestion
			return fmt.Errorf("unable to load provider CA file(s): %v", err)
```

Also, this method is meant to collate all of the possible errors and then return them as a single error at the end, can we make sure we retain that behaviour please